tests/test_siwe.py

```diff
@@ -48,21 +48,23 @@ def test_valid_message(self, test_name, test):
 
 
 class TestMessageValidation:
+    @pytest.mark.xfail  # It is expired.
     @pytest.mark.parametrize(
         "test_name,test",
         [(test_name, test) for test_name, test in validation_positive.items()],
     )
     def test_valid_message(self, test_name, test):
         siwe_message = SiweMessage(message=test)
-        siwe_message.validate() == True
+        siwe_message.validate()
 
     @pytest.mark.parametrize(
         "test_name,test",
         [(test_name, test) for test_name, test in validation_negative.items()],
     )
     def test_invalid_message(self, test_name, test):
-        siwe_message = SiweMessage(message=test)
-        siwe_message.validate() == False
+        with pytest.raises(Exception):
```

Comment:
nit: can we expect InvalidSignature here instead of a generic exception?

Reply:
Ha, I thought you couldn't use multiple exceptions in a `raises` but you can with a tuple. Fixed.